webapp/src/containers/Rhs/index.tsx

```diff
@@ -1,24 +1,49 @@
-import React from 'react';
+import React, {useEffect} from 'react';
 
 import usePluginApi from 'hooks/usePluginApi';
 import {getprojectDetailsState, getRhsState} from 'selectors';
 
+import LinearLoader from 'components/loader/linear';
+
+import plugin_constants from 'plugin_constants';
+
+import AccountNotLinked from './accountNotLinked';
 import ProjectList from './projectList';
 import ProjectDetails from './projectDetails';
 
 const Rhs = (): JSX.Element => {
     const usePlugin = usePluginApi();
 
+    // Fetch the connected account details when RHS is opened
+    useEffect(() => {
+        if (getRhsState(usePlugin.state).isSidebarOpen) {
```

Comment:
Why we're adding this condition? Won't this `useEffect` only run when the `RHS` is opened?

Reply:
this has been removed in later PRs